Server/controllers/time-entry.controller.js

```diff
@@ -4,13 +4,124 @@
  *
  * @class timeEntriesController
  */
+const mongoose = require('mongoose');
+const timeEntryModel = require('../models/time-entry.model');
+const dataLayer = require('../datalayer/time-entry.datalayer')
+const moment = require('moment');
+
+
 class timeEntriesController {
 
-    static getTimeEntriesByUserId (req, res) {}
+    //  GET 
+
+    static async getTimeEntriesByUserId(req, res) {
+        let timeEntries;
+
+        const obj = {
+            "userId": req.query.user,
+            "limit": req.query.limit
+        }
+
+        try {
+            timeEntries = await dataLayer.getTimeEntriesByUserId(obj);
```

Comment:
We should also check if userId is not undefined. If limit is undefined, we should set a default value say 1

Reply:
Updated as suggested

Reply:
We should check in controller itself, like controller should control the value it is passing to dataleyer.
In case user id is not present, it can be treated as bad request

Reply:
Updated